tedge/src/cli/connect/bridge_config_azure.rs

```diff
@@ -50,7 +50,7 @@ impl From<BridgeConfigAzureParams> for BridgeConfig {
             use_agent: false,
             try_private: false,
             start_type: "automatic".into(),
-            clean_session: true,
+            clean_session: false,
```

Comment:
This change is not related to the issue beyond the fact that it's related to the clean-session flag and could be moved in different PR.

Reply:
What's the impact of this change here to the cloud mappers? I'm assuming that this would make the sessions of the mappers persistent. But wasn't that the case before? If not, how did we not face this reported issue with the mappers earlier?

Reply:
This changes is related to the connection between the local MQTT bus and the remote MQTT end-point. With `clean_session: true`, one loses all the messages that have been sent to or received from the cloud when the local MQTT bus is down. Setting `clean_session: false`, these messages are persisted along the bridge session.